web/modules/custom/custom_weather/custom_weather.services.yml

```diff
@@ -0,0 +1,3 @@
+services:
+  Drupal\custom_weather\Service\UserCityHandler:
```

Comment:
Most times in Drupal we are setting a more meaningful name for the service, not just a class path.
Services could have overrides as well, so if anyone provides another implementation for that, name loses any sense